lib/chainweb/pact/local.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Kadena.Chainweb.Pact.Local do
+  @moduledoc """
+  Local endpoint implementation
+  """
+
+  alias Kadena.Chainweb.Request
+  alias Kadena.Chainweb.Pact.{LocalRequestBody, LocalResponse, Spec}
+  alias Kadena.Types.Command
+
+  @behaviour Spec
+
+  @endpoint "local"
+
+  @type cmd :: Command.t()
+  @type json :: String.t()
+
+  @impl true
+  def process(%Command{} = cmd, network_id: network_id, chain_id: chain_id) do
+    request_body = json_request_body(cmd)
+    headers = [{"Content-Type", "application/json"}]
```

Comment:
@EdwinGuayacan following the same standard names and order, it can be replaced with: 
```elixir
headers = [{"Content-Type", "application/json"}]
body = json_request_body(cmd)
```